README.md

```diff
@@ -1,30 +1,38 @@
+The purpose of this repository is to provide a home for several data tutorials written to improve the accesibility of the different GeoNet data sets. This repository is an easy way to access the tutorials, provides versioning and allows users to suggest changes or improvements.
 
-The tutorials in this repository are mostly [Jupyter notebook](https://jupyter.org/) files. They demonstrate some simple ways to retrieve and work with data from different GeoNet services such as: FDSN, FITS, Tilde, etc. Most are written in the Python programming language. Older versions of some notebooks were written in the R programming language. We are no longer supporting these, but the notebooks are still available, although we make no guarantee about their current usability. To access these notebooks, please use this [github commit](https://github.com/GeoNet/data-tutorials/tree/5609561894b924211da975d1794eb00b5fcff99d).
+The tutorials in this repository are mostly [Jupyter notebook](https://jupyter.org/) files. They demonstrate some simple ways to retrieve and work with data from different GeoNet services such as: FDSN, Tilde, etc. Most are written in the Python programming language. Older versions of some notebooks were written in the R programming language. We are no longer supporting these, but the notebooks are still available, although we make no guarantee about their current usability. To access these old R notebooks, please use this [github commit](https://github.com/GeoNet/data-tutorials/tree/5609561894b924211da975d1794eb00b5fcff99d).
 
 **All notebooks use Python 3. We do not support Python 2.7.**
 
-Tutorials are reviewed every 3 - 6 months. We confirm that they still run, and make any necessary adjustments so that they remain a valuable, working resource for GeoNet's data users.
+Tutorials are reviewed every 3 - 6 months, or sooner if circumstances require. During review, we confirm that notebooks still run, and make any necessary adjustments so that they remain a valuable, working resource for GeoNet's data users.
 
-Tutorials are organised by data access method, rather than data type. Within the folder for each data access method is a file README.md. This file contains most of the general material about data accessed by that method. This frees up individual notebooks to concentrate on data access and use, and reduces the maintenance required for each notebook. When you are using a particular notebook, it is therefore important that you refer to the README.md file in the same folder as the notebook.
+Tutorials are mostly organised by data access method. In cases where one broad type of data is available through several access methods or there are several different kinds of data, we may also provide tutorials for a specific data type.
```

Comment:
I think I get the sense (?) but struggling a bit on how the sentence is written. Perhaps this can be simplified as
```
Tutorials are mostly organised by data access method. Some data types are available through different access methods, and some access methods can be used to access several data types (i.e. Tilde). In those cases, we provide specific tutorials.
```
i'm not sure I got the sense of your sentence though!

Reply:
I agree mine isn't as clear as it should be, but I didn't think your was correct either. How about we just state was we have done (which is the main thing) and don't bother with the why we did it. `Tutorials are mostly organised by data access method, but in some cases we may also provide tutorials for a specific data type.`

Reply:
nice, thanks!! and agree, mine wasn't correct as I did not understand the message! nice resolution